cypress/integration/login_page.ts

```diff
@@ -0,0 +1,69 @@
+const clearForm = () => {
+  cy.get('input[id="birthDay"]')
+    .clear()
+    .get('input[id="birthMonth"]')
+    .clear()
+    .get('input[id="birthYear"]')
+    .clear();
+};
+
+describe('Login', () => {
+  it('shows error on empty values', () => {
+    cy.visit('http://localhost:3000/login');
+    cy.get('[type="submit"]').click();
+    cy.contains('The birthday should be dd.mm.yyyy.');
+  });
+
+  it('fill form but user is too old', () => {
+    cy.get('input[id="birthDay"]')
+      .type('01')
+      .get('input[id="birthMonth"]')
+      .type('01')
+      .get('input[id="birthYear"]')
+      .type('1950')
+      .get('input[id="birthDay"]');
```

Comment:
That was my plan, but because i'm struggling with setting up typescript for these tests, whole thing will crash when I need to specify types for helper function. (e.g `birthDay:string`